KVSS: Key-Value Storage Systems subsystem (#103244)
zephyrproject-rtos/zephyr: doc/releases/migration-guide-4.4.rst

  :dtcompatible:`zephyr,fstab,fatfs` with the ``automount`` property are present in the devicetree.
  Applications that do not want this behavior need to explicitly disable this option.

* NVS and ZMS have been moved to the new Key-Value Storage System (kvss); the move

>   has affected NVS and ZMS interface header paths that have been moved from

Reviewer: ```suggestion
  affects NVS and ZMS interface header paths which have been moved from
```
